tutorials/preprocessing/plot_40_artifact_correction_ica.py

```diff
@@ -101,6 +103,12 @@
 #     reconstruct using the 50 ICs plus the first 70 PCs in the PCA residual
 #     (numbers 51-120).
 #
+#     If you have previously been using **EEGLAB's** ``runica()`` and are
```

Comment:
For example this could be a different `sidebar` with a title `For EEGLAB users` or something